cartography/rules/data/rules/unmanaged_accounts.py

```diff
@@ -7,18 +7,21 @@
 # Facts
 _unmanaged_accounts_ontology = Fact(
     id="unmanaged-accounts-ontology",
-    name="User accounts not linked to a user identity",
-    description="Finds user accounts that are not linked to an ontology user node.",
+    name="Active Accounts Without User Identity Links",
```

Comment:
Let's use the wording you used below: "Active accounts without linked user identities"